### PR TITLE
Added latam currency

### PR DESCRIPTION
**Pull request by Jzarecta**
Added latam currencies

### CODE REVIEW

**Comment by teran-mckinney** [comment]
Have you tested to make sure all of these work in all of the rates APIs integrated?

**Comment by Jzarecta** [comment]
Just finalized moving around the exchange rates for the different currencies.

**Comment by teran-mckinney** [comment]
Thank you! And the rates API has all of these available?

File: bitcash/network/rates.py [hunk 1]
```diff
@@ -44,6 +44,14 @@
     ('sgd', 'Singapore Dollar'),
     ('thb', 'Thai Baht'),
     ('twd', 'New Taiwan Dollar'),
+    ('ḿxn', 'Mexican Peso'),
```
Comment (merc1er) [inline bitcash/network/rates.py]: Have you tested this currency? It's not working on my end.
Also it seems that special characters are not compliant with: https://en.wikipedia.org/wiki/ISO_4217

Comment (Jzarecta) [reply bitcash/network/rates.py]: There shouldn't be any special characters.

Comment (teran-mckinney) [reply bitcash/network/rates.py]: > ('ḿxn'

You can see the m.

Comment (teran-mckinney) [reply bitcash/network/rates.py]: Thank you!

**Comment by teran-mckinney** [comment]
@merc1er does this look okay to you?

**Comment by teran-mckinney** [comment]
Thank you @Jzarecta and @merc1er!

**Pull request merged**